src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,36 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+type UserWithTodo = {
```

Comment:
Let's extend  from existing type/interface

Reply:
types were moved from FC to files